### DISCUSSION TITLE
How to get rid of hotkey

### DISCUSSION THREAD

**Discussion by Beckfield** [openstyles/stylus#1305] [Q&A] [answered]
Out of curiosity, I created a hotkey for the color picker in the Stylus editor.  Now I can't find any way to get rid of it.  I tried the Delete key, but it just thinks I want to use the Delete key as a hotkey.  Nothing I've tried just gets rid of the existing hotkey.

The 'x' within the field that is mentioned in the [documentation](https://github.com/openstyles/stylus/wiki/Colorpicker) isn't there:
![image](https://user-images.githubusercontent.com/4588823/128437898-cf967b9d-4a0f-40f5-938f-510f5f7896ac.png)

How do I do this?

**Comment by tophf** [comment, marked as answer]
In Chrome there's a small `x` to clear the field. Evidently I forgot that Firefox doesn't show it. I'll add it explicitly for the next release. Meanwhile open any Stylus page, press Ctrl-Shift-I (don't do it in this hotkey input box though) to open `devtools`, switch to `console`, paste `prefs.set('editor.colorpicker.hotkey', '')`, press the <kbd>Enter</kbd> key, and reload the editor tab.

**Reply by Beckfield** [reply]
Thanks, that worked.

MS Edge and Vivaldi (both chromium-based) don't show the `x` either.
